node/src/aura_or_nimbus_consensus.rs

```diff
@@ -164,7 +168,7 @@ where
                             *timestamp,
                             slot_duration,
                         );
-                Ok((timestamp, slot))
+                Ok((slot, timestamp))
```

Comment:
This change looks odd.

Reply:
https://github.com/paritytech/substrate/pull/12224